locale/manageiq.pot

```diff
@@ -50536,6 +50628,11 @@ msgstr ""
 msgid "Not available. Did you assigned Cloud Provider and run SSA?"
 msgstr ""
 
+#: ../app/models/manageiq/providers/ibm_power_hmc/infra_manager/lpar.rb
+#: ../app/models/manageiq/providers/ibm_power_hmc/infra_manager/template.rb
+msgid "Not connected to ems"
```

Comment:
@agrare We probably shouldn't use EMS here as that's inside termonology?

Reply:
:+1: agreed in core we call this `"The VM is not connected to an active Provider` (from `supports :control`)

Reply:
https://github.com/ManageIQ/manageiq-providers-ibm_power_hmc/pull/131